### DISCUSSION TITLE
taro 3.6预计什么时候发布？

### DISCUSSION THREAD

**Discussion by airqj** [NervJS/taro#11758] [Q & A]
去年说是3.6版本会支持ssr,非常期待这个特性 😊

**Comment by Chen-jj** [comment]
3.5 会有结合 next 使用的特性，这一两周会发布的，详情可以关注发版的文章哈

**Reply by airqj** [reply]
非常期待 👍🏻

**Reply by thunur** [reply]
3.6会使用vite编译吗?

**Reply by Chen-jj** [reply]
> 3.6会使用vite编译吗?

支持 vite 涉及到整个编译链条的改造，估计要到下半年随 3.6 或 3.7 发布了
